fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1259,5 +1276,18 @@ public static String buildSearchSelfUri(String requestUriString, FHIRSearchConte
          */
         return UriBuilder.builder().context(context).requestUri(requestUriString).toSearchSelfUri();
     }
-
+    
+    
+    /**
+     * Return only the "text" element, the 'id' element, the 'meta' element, and only top-level mandatory elements
```

Comment:
extremely minor
```suggestion
     * Return only the "text" element, the 'id' element, the 'meta' element, and only top-level mandatory elements.
```

Reply:
I don't want to commit the suggested change by clicking the button, because this will force me to fix the sign-off issue via command line. so I have just made change locally and pushed to git via eclipse commit UI directly.

Reply:
FYI, you can manually add the `Signed-of-by: ` at the end of your commit message.  But doing it locally is really no problem either.